doc/help.rst

```diff
@@ -2,38 +2,98 @@
 Help
 ####
 
-You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods.
+You can use the **Help** pane to find, render and display rich documentation for any object with a docstring, including modules, classes, functions and methods. This allows you to access documentation easily directly from Spyder not having to break your workflow.
+
+|
+
+.. raw:: html
+
+	<iframe src="https://giphy.com/embed/H1quFSJ3LSwmVwvIhU" width="480" height="300" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/H1quFSJ3LSwmVwvIhU">via GIPHY</a></p>
```

Comment:
```suggestion
   <iframe src="https://giphy.com/embed/H1quFSJ3LSwmVwvIhU" width="480" height="300" frameBorder="0" class="giphy-embed" allowFullScreen></iframe>
```

I don't feel this works well as the hero as opposed to a GIF (or even static image) showing a help pane and an editor (or even console) side by side and showing getting help on an object, as the latter is a far more common use case than typing it directly into the pane, and users will be very frustrated if they try to type it in and it doesn't work by default since you haven't explained that yet. Instead, this GIF is far better used to replace the rather pointless static image `images/help/help-object-box.png` currently used below to describe this as I mention there.

This again needs all the formatting and fixes as the previous that I reviewed before you added this. Also, again, please be careful not to inject spurious tabs into these files, which can only cause problems. Your editor/IDE should be set accordingly.